tests/test_admin.py

```diff
@@ -370,3 +374,182 @@ def test_category_field_ordering_unversioned(self):
+        if not is_versioning_enabled():
+            alias1_content = alias1.get_content(language=self.language)
+            alias2_content = alias2.get_content(language=self.language)
+            alias3_content = alias3.get_content(language=self.language)
+            aliascontent1_url = admin_reverse(
+                CHANGE_ALIASCONTENT_URL_NAME, args=[alias1_content.pk]
+            )
+            aliascontent2_url = admin_reverse(
+                CHANGE_ALIASCONTENT_URL_NAME, args=[alias2_content.pk]
+            )
+            aliascontent3_url = admin_reverse(
+                CHANGE_ALIASCONTENT_URL_NAME, args=[alias3_content.pk]
+            )
+
+        self.assertContains(response, aliascontent1_url)
+        self.assertContains(response, aliascontent2_url)
+        self.assertContains(response, aliascontent3_url)
+
+
+class CategoryAdminViewsTestCase(BaseAliasPluginTestCase):
+
+    def test_changelist(self):
+        Category.objects.all().delete()
+        category1 = Category.objects.create()
+        category2 = Category.objects.create()
+        category1.translations.create(language_code='en', name='Category 1')
+        category2.translations.create(language_code='en', name='Category 2')
+        category1.translations.create(language_code='de', name='Kategorie 1')
+        category2.translations.create(language_code='fr', name='Catégorie 2')
+        category1.translations.create(language_code='it', name='Categoria 1')
+
+        with self.login_user_context(self.superuser):
+            with force_language('en'):
+                en_response = self.client.get(self.get_category_list_endpoint())
+            with force_language('de'):
+                de_response = self.client.get(self.get_category_list_endpoint())
+            with force_language('fr'):
+                fr_response = self.client.get(self.get_category_list_endpoint())
+            with force_language('it'):
+                it_response = self.client.get(self.get_category_list_endpoint())
+
+        self.assertContains(en_response, 'Category 1')
+        self.assertContains(en_response, 'Category 2')
+        self.assertNotContains(en_response, 'Kategorie 1')
+        self.assertNotContains(en_response, 'Catégorie 2')
+        self.assertNotContains(en_response, 'Categoria 1')
+
+        self.assertContains(de_response, 'Kategorie 1')
+        self.assertContains(de_response, 'Category 2')  # fallback
+        self.assertNotContains(de_response, 'Category 1')
+        self.assertNotContains(de_response, 'Catégorie 2')
+        self.assertNotContains(de_response, 'Categoria 1')
+
+        self.assertContains(fr_response, 'Category 1')  # fallback
+        self.assertContains(fr_response, 'Catégorie 2')
+        self.assertNotContains(fr_response, 'Category 2')
+        self.assertNotContains(fr_response, 'Kategorie 1')
+        self.assertNotContains(fr_response, 'Categoria 2')
+
+        self.assertContains(it_response, 'Catégorie 2')  # fallback
```

Comment:
```suggestion
        self.assertContains(it_response, 'Catégorie 2')
```

This isn't the fallback as per the comment I don't think, the fallback would be: "Category 2"

Reply:
In `test_settings.py` it defines `fr` as the fallback for `it`